test-storybooks/yarn-pnp/package.json

```diff
@@ -73,7 +74,7 @@
     "eslint-plugin-storybook": "9.1.0-alpha.6",
     "globals": "^16.2.0",
     "playwright": "1.52.0",
-    "storybook": "9.1.0-alpha.6",
+    "storybook": "file:../../code/core",
```

Comment:
I can't imagine this makes a difference, given the resolution above?

Reply:
it kinda makes no sense to even specify any version here at all.
It's just going to go out of date, and be confusing. I think the file-protocol makes it more clear, and won't go stale.